pkg/logs/logs.go

```diff
@@ -25,9 +25,41 @@ func NewLogsClient(kubernetesClient kclient.ClientInterface) *LogsClient {
 
 var _ Client = (*LogsClient)(nil)
 
+func (o *LogsClient) AllModeLogs(componentName string, namespace string) ([]map[string]io.ReadCloser, error) {
+	logs := []map[string]io.ReadCloser{}
+
+	devModeLogs, err := o.DevModeLogs(componentName, namespace)
+	if err != nil {
+		return nil, err
+	}
+	logs = append(logs, devModeLogs...)
+
+	deployModeLogs, err := o.DeployModeLogs(componentName, namespace)
+	if err != nil {
+		return nil, err
+	}
+	logs = append(logs, deployModeLogs...)
+
+	return logs, nil
+}
+
 func (o *LogsClient) DevModeLogs(componentName string, namespace string) ([]map[string]io.ReadCloser, error) {
 	// get all resources in the namespace which are running in Dev mode
 	selector := odolabels.Builder().WithComponentName(componentName).WithMode(odolabels.ComponentDevMode).Selector()
```

Comment:
You should use the `odolabels.GetSelector` function instead, which builds the "standard" selectors. Builder is for specific cases only

Reply:
Done.